stress/shell/src/main/java/alluxio/stress/cli/fuse/FuseIOBench.java

```diff
@@ -75,68 +84,114 @@ public String getBenchDescription() {
+        "Note that \"--operation\" is required, \"--local-path\" can be a local filesystem "
+            + "path or a mounted Fuse path, and non-cluster mode only supports local read.",
         "",
         "Example:",
+        "# The test will be run in cluster mode using job service",
         "# The test data will be written to /mnt/alluxio-fuse/FuseIOTest",
         "# Files will be evenly distributed into 32 directories, each contains 10 files of "
-            + "size 100 MB",
+            + "size 100 MB. 32 threads of each worker will be used to generate the files",
         "# Metadata of the test files will be cached",
-        "# 32 threads will be used for writing the data, and 16 threads will be used for "
-            + "testing the reading throughput",
+        "# 16 threads of each worker will be used for testing the reading throughput with "
+            + "ClusterRead.",
         "# 5 seconds of warmup time and 30 seconds of actual reading test time",
         "$ bin/alluxio runClass alluxio.stress.cli.fuse.fuseIOBench --operation Write \\",
         "--local-path /mnt/alluxio-fuse/FuseIOTest --num-dirs 32 --num-files-per-dir 10 \\",
-        "--file-size 100m --threads 32",
+        "--file-size 100m --threads 32 --cluster",
         "$ bin/alluxio runClass alluxio.stress.cli.fuse.fuseIOBench --operation ListFile \\",
         "--local-path /mnt/alluxio-fuse/FuseIOTest",
         "$ bin/alluxio runClass alluxio.stress.cli.fuse.fuseIOBench --operation Read \\",
         "--local-path /mnt/alluxio-fuse/FuseIOTest --num-dirs 32 --num-files-per-dir 10 \\",
-        "--file-size 100m --threads 16 --warmup 5s --duration 30s",
+        "--file-size 100m --threads 16 --warmup 5s --duration 30s --cluster",
         ""
     ));
   }
 
   @Override
   public void prepare() throws Exception {
+    if (mBaseParameters.mCluster) {
+      // For cluster mode, this function is called once before the job is submitted to the job
+      // service. Nothing should be done, otherwise the bench will break.
+      return;
+    }
     if (mParameters.mThreads > mParameters.mNumDirs) {
       throw new IllegalArgumentException(String.format(
           "Some of the threads are not being used. Please set the number of directories to "
               + "be at least the number of threads, preferably a multiple of it."
       ));
     }
+    File localPath = new File(mParameters.mLocalPath);
+
     if (mParameters.mOperation == FuseIOOperation.WRITE) {
       LOG.warn("Cannot write repeatedly, so warmup is not possible. Setting warmup to 0s.");
       mParameters.mWarmup = "0s";
       for (int i = 0; i < mParameters.mNumDirs; i++) {
-        Files.createDirectories(Paths.get(mParameters.mLocalPath + "/" + i));
+        Files.createDirectories(Paths.get(String.format(
+            "%s/%s/dir-%d", mParameters.mLocalPath, mBaseParameters.mId, i)));
+      }
+    } else {
+      if ((mParameters.mOperation == FuseIOOperation.REMOTE_READ
```

Comment:
Put parameters check in the main function instead of in the prepare stage.
It doesn't belong to prepare(), it's an issue with wrong inputs

Reply:
It hasn't parsed the parameters yet at main() time. The prepare function is the earliest time after the parameters get parsed.